### PR TITLE
Provisioning guide

### PR DESCRIPTION
**Pull request by rahulbajaj0509**
This pr contains the following:
* New variables
* Few guidelines that we must follow while reviewing/creating other guides
* Changed and added few contents from the upstream guide.

### CODE REVIEW

**Comment by rahulbajaj0509** [comment]
Due to some git issues, I had to delete the previous branch, there this is a new pull request. 
Referencing the previous one here: https://github.com/theforeman/foreman-documentation/pull/7 

@lzap let get this merged, thanks for reviewing @melcorr and @ewoud :)

**Comment by melcorr** [comment]
@ rahulbajaj0509 you've added back in a file that was deleted as part of another pull request:

 guides/doc-Provisioning_Guide/topics/Containers_configuring_atomic_host.adoc

**Comment by rahulbajaj0509** [comment]
@melcorr nice observation, let me fix that and then we can merge this!

**Comment by melcorr** [comment]
Please hurry with this so that we can make progress and not have wasted an additional two weeks on this.

File: guides/doc-Provisioning_Guide/topics/Networking.adoc [hunk 1]
```diff
@@ -523,6 +551,11 @@ To configure the {SmartProxy} to chainboot iPXE, complete the following steps:
 # cp /usr/share/ipxe/undionly.kpxe /var/lib/tftpboot/undionly-ipxe.0
 ----
 +
+ifeval::["{build}" == "foreman"]
+[NOTE]
+On systems with SELinux, perform the following command.
+
+endif::[]
```
Comment (lzap) [inline guides/doc-Provisioning_Guide/topics/Networking.adoc]: Extra line but let's just not block this. That's fine.

**Comment by rahulbajaj0509** [comment]
Thanks @lzap!

**Pull request merged**